perma_web/perma/admin.py

```diff
@@ -542,10 +543,6 @@ def __init__(self, *args, **kwargs):
 
         super(LinkUserChangeForm, self).__init__(*args, **kwargs)
 
-        # make sure that user's current organizations show even if they have been deleted
-        self.initial['organizations'] = Organization.objects.all_with_deleted().filter(users__id=kwargs['instance'].pk)
-        self.fields['organizations'].queryset = Organization.objects.all_with_deleted()\
-            .filter(Q(user_deleted=False)|Q(pk__in=self.initial['organizations']))
```

Comment:
I wonder if we need to re-implement something like this, for the UserOrganizationAffiliation inline. Do you know?

The way to test would be, create a new organization, add a user to that organization, then soft-delete the organization using this button on `/manage/organizations`, and see if the affiliation still shows up in the admin's inline.

![image](https://github.com/user-attachments/assets/f33d7b95-47da-49f6-bf00-786d7446407d)

Reply:
Yep, I tested that flow in the admin user change form. And I did this through the init method in the UserOrganizationAffiliationAdminForm class. Here they are:
![image](https://github.com/user-attachments/assets/af782cb9-6efa-4e1e-904c-327f73095d39)
![image](https://github.com/user-attachments/assets/ed36d820-229c-4d2c-9efd-75c18bb2e7bf)
To

Reply:
sweet!